### PR TITLE
Fix broken doctest

### PR DESCRIPTION
**Pull request by fingolfin**

### CODE REVIEW

**Comment by fingolfin** [comment]
I did this before in PR #35 which you merged ... and then reversed... This time I added a comment to docs/make.jl that explain how it works. I hope this time it'll stick.

And maybe next time ask questions if  you can't figure out how something works, it's not as if our offices were, like, next to each other :-). We also have plenty other people who know about Julia and can help. And we have a Slack (which you have joined before, I think? Otherwise see <https://oscar-system.org/slack> ) which is a great place to get advise. And then of course that's just OSCAR specific, there is a Julia Slack, discourse, and many more resources were you can learn and get support.

**Review by blociss — APPROVED** [review]

**Comment by fingolfin** [comment]
I find your actions very confusing especially since you almost never explain them :-(. Like now: you approve but then don't merge but instead close the PR? Why?

in the meantime you changed the CI code to use `docs/makelocal.jl` -- clearly this is incompatible with this PR. It also is odd to me, because that file, as its name suggests, was/is for local use, not on a (remote) CI server

**Pull request closed without merging**
